nmrfx-bom/pom.xml

```diff
@@ -19,6 +19,12 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.nanalysis</groupId>
+                <artifactId>spinlab-dataset</artifactId>
+                <version>1.1.0</version>
```

Comment:
Repository for this project is here: https://github.com/nanalysis/spinlab-dataset